### PR TITLE
Add Nicos favorite poem

### PR DESCRIPTION
**Pull request by davewragg1**

### CODE REVIEW

File: Nico.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Roses are red
+violets are blue
```
Comment (PatricHolmvall) [inline Nico.txt:2]: Nice, I like violets

File: Nico.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+poems are stupid
+and so are you
```
Comment (davewragg1) [inline Nico.txt:4]: You saw right through me.

Comment (NicolaiHaaberJunge) [inline Nico.txt:4]: :(

File: Nico.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Roses are red
+violets are blue
+poems are stupid
```
Comment (davewragg1) [inline Nico.txt:3]: so true

**Pull request merged**